site/plugins/search/src/Entry.php

```diff
@@ -128,18 +128,22 @@ protected function templates(): array
 	 */
 	public function toData(): array
 	{
-		$data = ['objectID' => $this->page->id()];
+		$data             = [];
+		$data['objectID'] = match ($this->template()) {
+			'plugin' => $this->page->url(),
+			default  => $this->page->id()
+		};
```

Comment:
`Suggestion:` I would keep `objectID` always with an id and for all entries also adding the URL as `url` separately. I think that'll be helpful long term. We only need to change the frontend part then to use `url`.

Reply:
I've reverted the change. Btw `/plugins/developer/plugin-name` uri automaticly already redirected to new subdomain 🎉 I wasn't aware it.

https://getkirby.com/plugins/tobimori/seo

So no need to handle from JS side.